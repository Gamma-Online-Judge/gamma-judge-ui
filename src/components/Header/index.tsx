import Latex from 'react-latex-next'

const Header = () => {
    return (
        <div className="jumbotron text-center bg-dark text-white">
            <h1><Latex>{'$\\gamma$'}</Latex> Online Judge </h1>
            <p>Problemas e eventos de programação competitiva</p>
        </div>
    );
};
export default Header;