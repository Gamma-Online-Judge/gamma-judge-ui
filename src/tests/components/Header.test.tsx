import renderer from 'react-test-renderer';
import Header from '../../components/Header';

test('Header snapshot', async () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
