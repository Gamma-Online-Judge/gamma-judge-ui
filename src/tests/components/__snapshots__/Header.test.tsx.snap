// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<nav
  className="navbar navbar-expand navbar-dark bg-green"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Gamma Online Judge
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="me-auto navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        className="nav-link"
        data-rr-ui-event-key="/"
        disabled={false}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="nav-link"
        data-rr-ui-event-key="/contests"
        disabled={false}
        href="/contests"
        onClick={[Function]}
      >
        Contests
      </a>
      <a
        className="nav-link"
        data-rr-ui-event-key="/problems"
        disabled={false}
        href="/problems"
        onClick={[Function]}
      >
        Problems
      </a>
    </div>
  </div>
</nav>
`;
